ggshield/utils/git_shell.py

```diff
@@ -7,11 +7,13 @@
 from shutil import which
 from typing import Dict, List, Optional, Union
 
+from ggshield.utils.os import getenv_int
+
 
 EMPTY_SHA = "0000000000000000000000000000000000000000"
 EMPTY_TREE = "4b825dc642cb6eb9a060e54bf8d69288fbee4904"
 
-COMMAND_TIMEOUT = 45
+COMMAND_TIMEOUT = getenv_int("GG_GIT_TIMEOUT", 45)
```

Comment:
Should we keep a `DEFAULT_COMMAND_TIMEOUT = 45` to highlight the fact that this is a constant ?

Reply:
I don't think it's worth it, unless we use `DEFAULT_COMMAND_TIMEOUT` somewhere else.